site/docs/concepts/connectors.md

```diff
@@ -371,6 +371,62 @@ You then use this `config.yaml` within your Flow catalog.
 Flow looks for and understands the `encrypted_suffix`,
 and will remove this suffix from configuration keys before passing them to the connector.
 
+### Connecting to endpoints on secure networks
+
+In some cases, your source or destination endpoint may be within a secure network, and you may not be able
+to allow direct access to its port due to your organization's security policy.
+
+[SHH tunneling](https://www.ssh.com/academy/ssh/tunneling/example#local-forwarding), or port forwarding,
+provides a means for Flow to access the port indirectly through an SSH server.
+
+To set up and configure the SSH server, see the [guide](../../guides/connect-network/).
+
+:::info Beta
+Currently, Flow supports SSH tunneling on a per-connector basis; consult the appropriate connector's documentation
+to verify. Estuary plans to expand this to universally cover all connectors in the future. Additionally,
+we'll add support for other means of secure connection.
+:::
+
+After verifying that the connector is supported, you can add a `proxy` stanza to the capture or materialization
+definition to enable SSH tunneling.
+
+```yaml title="postgres-ssh-tunnel.flow.yaml"
+captures:
+  acmeCo/postgres-ssh:
+    endpoint:
+      connector:
+        image: ghcr.io/estuary/source-postgres:dev
+        config:
+          # When using a proxy like SSH tunneling, set to localhost
+          host: localhost
+          # 0 allows Flow to dynamically choose an open port on your local machine
+          # to connect to the proxy (recommended).
+          # If necessary, you're free to specify a different port.
+          port: 0
+          database: flow
+          user: flow_capture
+          password: secret
+          proxy:
+            # Support for other proxy types will be enabled in the future.
+            proxy_type: ssh_forwarding
+              ssh_forwarding:
+                # Location of the remote SSH server that supports tunneling.
+                ssh_endpoint: 198.21.98.1
```

Comment:
`ssh_endpoint` is in the form of `ssh://hostname[:port]`.
So this could be changed to `ssh://98.21.98.1`.